app/views/aid_applications/confirmations/edit.html.erb

```diff
@@ -0,0 +1,83 @@
+            <div class="accordion__content" id="document_proof_of_address">
+              <p><%= t('.verifications.proof_of_address_accordion.body_html') %></p>
+            </div>
+          </div>
+          <div class="accordion">
+            <button class="accordion__button" aria-expanded="true" aria-controls="document_covid19_impact"><%= t('.verifications.covid19_impact_accordion.title') %></button>
+            <div class="accordion__content" id="document_covid19_impact">
+              <p><%= t('.verifications.covid19_impact_accordion.body_html') %></p>
+            </div>
+          </div>
+
+          <hr>
+
+          <h2><%= t('.application_number.title') %></h2>
+
+          <h3 class="notice text--centered">
+            <%= @aid_application.application_number %>
+          </h3>
+          <p class="with-padding-med"><em><%= t('.application_number.confirm') %></em></p>
+          <%= link_to(edit_organization_aid_application_applicant_path(current_organization, @aid_application), class: 'button button--small button--secondary ') do %>
+            <i class="button__icon--left icon-arrow_back"></i>
+            <%= t('.application_number.edit_contact') %>
+          <% end %>
+          <%= f.cfa_checkbox :contact_method_confirmed, t('.application_number.contact_method') %>
+
+          <hr>
+
+          <h2><%= t('.next_steps.title') %></h2>
+          <ol class="list--numbered">
+            <li><%= t('.next_steps.step1') %></li>
+            <li><%= t('.next_steps.step2') %></li>
+            <li><%= t('.next_steps.step3') %></li>
+            <li><%= t('.next_steps.step4') %></li>
+            <li><em><%= t('.next_steps.step5') %></em></li>
+          </ol>
+
+            <%= f.cfa_radio_set(
+                    :card_receipt_method,
+                    label_text: t('.receive_card.title'),
+                    help_text: t('.receive_card.subtitle'),
+                    collection: [
+                        {value: AidApplication::CARD_RECEIPT_PICK_UP, label: t('.receive_card.options.pick_up')},
+                        {value: AidApplication::CARD_RECEIPT_DELIVER, label: t('.receive_card.options.deliver')},
+                        {value: AidApplication::CARD_RECEIPT_MAIL, label: t('.receive_card.options.mail')},
+                        {value: AidApplication::CARD_RECEIPT_DECIDE_LATER, label: t('.receive_card.options.decide_later')}
+                    ])
+            %>
+        </div>
+
+        <div class="form-card__footer">
+          <div class="toolbar">
+            <div class="toolbar__left"></div>
+            <div class="toolbar__right">
+              <%= f.button "Submit", type: 'submit', class: 'toolbar__item button button--primary' %>
+            </div>
+          </div>
+        </div>
+      <% end %>
+    </div>
+  </div>
+<% else %>
```

Comment:
I think we can remove this if/else because now we shouldn't be able to get to the confirmation page if the app is not yet submitted.

Reply:
Done

Reply:
Is it possible to move the form (or some major elements of it) into a shared partial that both Application and Verification controllers will share? I worry that the two will get out of sync very quickly.

Reply:
Whoops, oops. This is a different page altogether.